docs/Extensions/CustomWebSearch/README.md

```diff
@@ -0,0 +1,25 @@
+# Custom Web Search Extension
+
+This extension allows define custom search engines, to directly search on the preferred website.
+
+![Example](example.png)
+
+## Settings
+
+-   **Name**: The name for your custom search engine (only visual).
+-   **Prefix**: The prefix, which should trigger the custom search engine
+-   **Url**: The URL, to which you want to be redirected. Use `{{query}}` placeholder, where the search term should be inserted
+-   **Encode search term**: Specifies, if the search term should be URL encoded (e.g. `%20` for a space)
+
+![Settings](settings.png)
+
+## About this extension
+
+Author: [NiewView](https://github.com/NiewView)
+Extension Icon: Designed by [OpenMoji](https://openmoji.org/) – the open-source emoji and icon project. License: [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/#)
```

Comment:
Please use bullet points here:

```suggestion
- Author: [NiewView](https://github.com/NiewView)
- Extension Icon: Designed by [OpenMoji](https://openmoji.org/) – the open-source emoji and icon project. - 
- License: [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/#)
```

Reply:
I made a minor tweak to your suggestion, because the license only belongs to the Icon not the whole extension.